### PR TITLE
Phone verification in laravel using twilio

### PR DESCRIPTION
**Pull request by philzy94**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com)
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

File: content/articles/phone-verification-in-laravel-using-twilio/index.md [hunk 1]
```diff
@@ -0,0 +1,803 @@
...
+- You are having a [voice-enabled Twilio phone number]([url](https://www.twilio.com/console/phone-numbers/incoming))
+
+### Table of Contents
+- Creating and Setting Up a New Laravel App
+- Set Up Authentication (Controller, Routes and Views)
+- Database Migration
+- Set Up Twilio
+- Calling User's Phone Number
+- Instructing, Collecting Input and Account Verification
+- Conlusion
```
Comment (jumamiller) [inline content/articles/phone-verification-in-laravel-using-twilio/index.md:26]: Ensure you fo

**Review by jumamiller — CHANGES_REQUESTED** [review]
Ensure that you follow the set guidelines for an article on [this](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md) link.

Ensure all you code snippets are well explained!

**Comment by philzy94** [comment]
@jumamiller the article has been updated as you demanded

**Comment by ahmadmardeni1** [comment]
Upon running your article through our 3rd party plagiarism checker it seemed to raise a few flags and the % was higher than we typically accept.
Please see attached PDF - and revisit the article to ensure we are contributing wholly unique and original content.
Be sure to see our resources page to see more info on plagiarism and what is considered as such.
[Phone verification in laravel using twilio#5014](https://github.com/section-engineering-education/engineering-education/files/7553813/New.Scan.10_27.AM.Copyleaks.Report.pdf)

**Comment by philzy94** [comment]
@ahmadmardeni1 I will revisit the article but from the guide, we were asked to use quetext which I used and got less than 10% plagiarism.

**Comment by ahmadmardeni1** [comment]
We are not using Quetext anymore; we are using CopyLeaks but you can use any plagiarism checker. @philzy94

**Comment by philzy94** [comment]
> We are not using Quetext anymore; we are using CopyLeaks but you can use any plagiarism checker. @philzy94

@ahmadmardeni1 ok sir. I have done some modifications to my article, especially the code since that is where the plagiarism is more.

**Comment by philzy94** [comment]
@ahmadmardeni1 all necessary editing has been made

File: content/articles/phone-verification-in-laravel-using-twilio/index.md [hunk 2]
```diff
@@ -0,0 +1,1916 @@
+# Phone Verification Laravel Using Twilio
+
+
+
+
+
```
Comment (jumamiller) [inline content/articles/phone-verification-in-laravel-using-twilio/index.md:6]: What's this spacing for?

File: content/articles/phone-verification-in-laravel-using-twilio/index.md [hunk 3]
```diff
@@ -0,0 +1,1916 @@
+# Phone Verification Laravel Using Twilio
+
```
Comment (jumamiller) [inline content/articles/phone-verification-in-laravel-using-twilio/index.md:2]: The article lacks introduction part!

File: content/articles/phone-verification-in-laravel-using-twilio/index.md [hunk 4]
```diff
@@ -0,0 +1,1916 @@
...
+- You are having a[ Composer globally installed](https://getcomposer.org/doc/00-intro.md).
+
+- You are having[ Ngrok set up](https://ngrok.com/)
+
+- You are having [Twilio ](https://www.twilio.com/)account.
+
+- You are having a verified number (If you are in trial mode).
+
+- You are having a [voice-enabled Twilio phone number]([url](https://www.twilio.com/console/phone-numbers/incoming))
+
```
Comment (jumamiller) [inline content/articles/phone-verification-in-laravel-using-twilio/index.md:52]: Kindly avoid this repetitions 'you are having', look for a better alternative!

File: content/articles/phone-verification-in-laravel-using-twilio/index.md [hunk 5]
```diff
@@ -0,0 +1,1916 @@
...
+
+
+In this article, we will learn how to call a user's registered phone number to verify the phone number's authenticity. If the user answers the phone, he or she will be asked to enter a one-time password (OTP) that was generated and displayed on his or her screen during the signup process. The user account will not be verified if an incorrect OTP is entered or if the user refuses to answer the phone. The user account will be verified if the user answers the phone and enters the correct OTP.
+
+
+
+This phone number verification will be accomplished in Laravel by utilizing a Twilio programmable voice service called TwiML Voice: `<Gather>`, which allows you to gather a user's input while they are on the phone. Twilio is a cloud communication platform that provides a variety of products and solutions to help you engage your users, one of which is the Programmable Voice, which allows you to make and receive calls from within your application.
+
+
```
Comment (jumamiller) [inline content/articles/phone-verification-in-laravel-using-twilio/index.md:67]: Introduction within article's body?

File: content/articles/phone-verification-in-laravel-using-twilio/index.md [hunk 6]
```diff
@@ -0,0 +1,1916 @@
...
+### Creating and Setting Up a New Laravel App
+
+We'll use composer to create a new Laravel installation. We do this by entering the following command into our terminal: 
+
+
+
+```
+
+composer create-project laravel/laravel programmable-voice
+
```
Comment (jumamiller) [inline content/articles/phone-verification-in-laravel-using-twilio/index.md:78]: Is this supposed to be a command/ sentence?? bash? php? Laravel? Use appropriate markdown commands!

**Review by jumamiller — CHANGES_REQUESTED** [review]
Kindly remove all spacings( I've given an example above). Then, restructure the article to have the intro on top!

**Comment by hectorkambow** [comment]
@philzy94

**Comment by hectorkambow** [comment]
@philzy94 
 it seems this PR is still quite a bit aways from being completed. 
 
As a rule of thumb - please only submit articles (pull requests) with a complete and fully polished article - ready to be published. 
 
All revisions and edits should be completed on your own forked repo (as to not take up room in the queue). 
 
We will be closing this and please re-submit when it is ready to be published as a new PR (and you can add the duplicate label) and ping the reviewer and myself. 

Closing a PR does NOT mean the article can not be published (or fixed) it simply means that the edits, revisions, and fixes will happen outside of the queue.
 
👍 
cc @jumamiller

**Pull request closed without merging**